govcr.go

```diff
@@ -270,8 +532,23 @@ type RequestFilterFunc func(http.Header, []byte) (*http.Header, *[]byte)
 // Return values:
 //  - value 1 - Response's amended header
 //  - value 2 - Response's amended body
+// Deprecated: Use ResponseFilterFunc instead.
```

Comment:
If we're going to go down this road, let's get rid off `ResponseFilterFunc` altogether.
This would form v3 of govcr.

Reply:
👍 Just didn't want to break it unless you were fine with it.